gui/wxpython/modules/colorrules.py

```diff
@@ -822,10 +822,7 @@ def CreateColorTable(self, tmp=False):
 
         cmd = cmdlist_to_tuple(cmd)
         ret = RunCommand(cmd[0], **cmd[1])
-        if ret != 0:
-            return False
-
-        return True
+        return bool(ret == 0)
```

Comment:
When I added it (or ruff most of the time), here and elsewhere, (I added maybe 8 or so), it was when pyright was mentioning that the return type was possibly a bool, but also a long list of other things. It was the RunCommand that wasn't clear what the return type was. So, to make sure that it will really be a bool, I made sure to add bool(...) when the function I changed didn't have the same typing garanties.

Reply:
Just thought that `ret == 0` will return a _bool_ anyway, so no need for `bool(ret == 0)`

Reply:
So you still ask for the bool to be removed (and have type checkers not know for sure if it's a bool), or we keep it?

Reply:
Sorry for not being clear. I am all fine with bool() here. It just seemed inconsistent to me as bool() was not used in other, identical cases `return ret == 0` above.
So I thougt the pattern was to use bool() in case of multiple tests and no bool() in case of single tests. But I am fine with either, as long as it is consistent. And if bool() makes things more clear for checkers, that is a good argument to have it...

Reply:
Kept bool() when ambiguous.